examples/todo/rust/lib.rs

```diff
@@ -170,6 +170,10 @@ impl SerializedState {
 
 #[test]
 fn press_add_adds_one_todo() {
+    if std::env::var_os("SLINT_EMIT_DEBUG_INFO").unwrap_or_default() != "1" {
```

Comment:
```suggestion
    if option_env!("SLINT_EMIT_DEBUG_INFO").unwrap_or_default() != "1" {
```
What counts is what is set when being build.



(Note that unfortunately the println!() will not be seen since it is always capture :-()